aws_jobs/cyhy-data-extract.py

```diff
@@ -26,16 +25,19 @@
 from ConfigParser import SafeConfigParser
 from datetime import datetime
 from dateutil.relativedelta import relativedelta
+import dateutil.tz as tz
 from docopt import docopt
+import json
+import os
 import boto3
```

Comment:
You need a blank line before `import boto3` as a break between the built-in imports and the 3rd-party ones.